### DISCUSSION TITLE
<C-H> switching windows

### DISCUSSION THREAD

**Discussion by gemvcnte** [LazyVim/LazyVim#3273] [General]
i have a problem when i update my nvim and the lazy in my lazyvim. whenever i press <C-H> to go to the left window it's not working, but my <C-L> is working to go to the right window.

**Comment by folke** [comment]
Something probably overwrite the keymap.

Check with `:verbose nmap <c-H>`

**Reply by gemvcnte** [reply]
![image](https://github.com/LazyVim/LazyVim/assets/111511069/73381bea-715a-4b82-b223-ba4b083feea3)


here is the output in the messages of the lazyvim

**Reply by gemvcnte** [reply]
it seems okay but when i <C-H> in the nerd tree it keeps going back 1 directory until it reach the user directory.


edit: okay i think it's disregarding the <C-H> keymap because it's working when i do <C-W>h
